### DISCUSSION TITLE
Possibility to use data types (selectors, labels, etc.) in header attributes (frontmatter) directly in each file (not from the table)

### DISCUSSION THREAD

**Discussion by lorensansol** [RafaelGB/obsidian-db-folder#858] [Ideas]
Hello! Congratulations and thanks for the DB Folder plugin, it's amazing!

I think it would be great to have the ability to use data types (selectors, labels, etc.) in header attributes (frontmatter) directly in each file (not from the table), just like it's possible in Notion.

Perhaps with file templates it could be achieved.

**Comment by RafaelGB** [comment]
Hello @lorensansol , there is an awesome plugin with the purpose. Try https://github.com/chhoumann/MetaEdit
